src/core/reducers/survey.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+const DISMISS_SURVEY: 'DISMISS_SURVEY' = 'DISMISS_SURVEY';
+const SHOW_SURVEY: 'SHOW_SURVEY' = 'SHOW_SURVEY';
+
+export type SurveyState = {|
+  wasDismissed: boolean,
+|};
+
+export const initialState = {
+  wasDismissed: false,
+};
+
+type DismissSurveyAction = {|
```

Comment:
It's consistent throughout the file, and it's a nice short name, but at first glance it seems like these actions affect a "Survey", whereas they really affect a `SurveyNotice`. We're not showing and dismissing a survey, we're showing and dismissing a notice. Maybe the names should be changed to be clearer, or maybe that just adds unnecessary letters.

Reply:
The end result is still a survey so I think it makes sense.